server/routes/api/user.ts

```diff
@@ -295,6 +295,10 @@ userRoutes.route("/status").post(isAdmin, uploadHandler.any(), async (request, r
 	}
 	else if (status === "accepted") {
 		user.accepted = true;
+		let applicationBranch = (await Branches.BranchConfig.loadBranchFromDB(user.applicationBranch)) as Branches.ApplicationBranch;
+		user.confirmationDeadlines = ((await Branches.BranchConfig.loadAllBranches("Confirmation")) as Branches.ConfirmationBranch[])
+				.filter(c => c.usesRollingDeadline)
```

Comment:
@bunsenmcdubbs Please roll this into one call of `.filter()` for better performance